src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -131,10 +131,12 @@ interfaces:
         - shelf
         - books
         - edition
+        - review_copy
```

Comment:
A similar update has not been done to `src/test/java/com/google/api/codegen/testdata/library_gapic.yaml`. Also, isn't it a problem for oneof fields to be required, since only one of them can be populated?